src/core/App.tsx

```diff
@@ -1,8 +1,28 @@
-import React from 'react';
+import { ThemeProvider } from '@emotion/react';
+import Header from 'modules/components/Header';
+import { SideMenu } from 'modules/components/SideMenu';
+import { useEffect } from 'react';
+import { useAppDispatch } from 'shared/hooks';
+import { getAlbum } from 'shared/store/reducers/albumReducerSlice';
+import { getPhoto } from 'shared/store/reducers/photoReducerSlice';
+import { theme } from 'styles/theme';
 
 const App = (): JSX.Element => {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
```

Comment:
I recommend making those reequests on according pages for more optimized performance